sound/soc/sof/intel/hda.h

```diff
@@ -228,6 +227,10 @@
 #define HDA_DSP_REG_HIPCIE		(HDA_DSP_IPC_BASE + 0x0C)
 #define HDA_DSP_REG_HIPCCTL		(HDA_DSP_IPC_BASE + 0x10)
 
+/* Intel Vendor Specific Regsiters */
+#define HDA_VS_EM2			0x1030
+#define HDA_VS_EM2_L1SEN		BIT(13)
```

Comment:
This should be HDA_VS_INTEL_EM2. Anything vendor-specific needs to have a non-ambiguous owner.